README.md

```diff
@@ -1 +1,245 @@
+@property (nonatomic) CGFloat volume;
+
+@property (nonatomic) BOOL stereo;
+
+@property (nonatomic) CGFloat fmPreset1;
+@property (nonatomic) CGFloat fmPreset2;
+@property (nonatomic) CGFloat amPreset1;
+@property (nonatomic) CGFloat amPreset2;
+@property (nonatomic) CGFloat amPreset3;
+
+@end
+```
+
+## The Anatomy Of A Property
+
+Using the `@property` syntax, while it can feel verbose, is actually  a shorthand for generating the three components of a property: the instance variable, the "setter" method, and the "getter" method.
+
+#### The Instance Variable
+
+The **instance variable** (also "ivar") is the actual value that the property manages. It is both standard convention and implicit syntax that the instance variable is name after the property, but prepends an underscore character (`_`) for disambiguation. So, `_length` is the *instance variable* managed by the `length` property, and `_count` is the *instance variable* managed by the `count` property.
+
+**Note:** *Do* ***not*** *access an instance variable directly unless writing an initializer (common) or overriding a setter or getter (advanced). It best practice to use dot notation which implicitly calls the setter and getter methods for that property.*
+
+#### The Setter Method
+
+The **setter** is a method that is responsible for *writing to* (or "setting") the instance variable. It is both convention and implicit syntax that the setter is named after the property and prefixed with the word "set". The setter for the `length` property would be called `setLength:`, and the setter for the `count` property would be called `setCount:`. However, dot notation implicitly calls the setter method when used on the left side of an assignment operator:
+
+```objc
+// good style
+
+self.length = 10;
+```
+This is equivalent to:
+
+```objc
+// bad style
+
+[self setLength:10];
+```
+
+#### The Getter Method
+
+The **getter** is a method that is responsible for *reading from* (or "getting") the instance variable. It is both convention and implicit syntax that the getter is named directly after the property whose value it retrieves (though some `BOOL` properties prepend the word "is" to the getter). The getter for the `length` property would be a method called `length`, and the getter for the `count` property would be called `count`. Dot notation implicitly calls the getter method whenever it is submitted as an argument, or is used on the right side of an assignment operator:
+
+```objc
+// good style
+
+NSUInteger thisLength = self.length;
+[mutableArray addObject:@(self.length) ];
+```
+These are equivalent to:
+
+```objc
+// bad style
+
+NSUInteger thisLength = [self length];
+[mutableArray addObject:@([self length]) ];
+```
+
+Some of the methods which we have used in previous examples are actually the getter methods for properties:
+
+```objc
+// using getter methods (valid)
+
+NSString *welcome = @"Welcome to the Flatiron School!";
+NSString *welcomeUC = [welcome uppercaseString];
```

Comment:
For the sake of comparing this code directly to what follows, I'd skip this assignment in this code and just `NSLog` the whole shebang